cluster-autoscaler/go.mod

```diff
@@ -263,3 +262,5 @@ replace k8s.io/dynamic-resource-allocation => k8s.io/dynamic-resource-allocation
 replace k8s.io/kms => k8s.io/kms v0.29.0-alpha.3
 
 replace k8s.io/endpointslice => k8s.io/endpointslice v0.29.0-alpha.3
+
+replace github.com/satori/go.uuid => github.com/gofrs/uuid/v5 v5.0.0
```

Comment:
if you are removing the usage of `/satori`, why do we need the replace?

Reply:
satori replace is deleted, thank you